db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/OPENSTACK.class/compute_instance_resume_end.yaml

```diff
@@ -8,6 +8,8 @@ object:
     inherits: 
     description: 
   fields:
+  - rel3:
+      value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=prevent_retired_vm_start&param="
```

Comment:
@lfu 
Can we change the policy event to vm_resume?
/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_resume&param=